.pytool/Plugin/LineEndingCheck/Readme.md

```diff
@@ -0,0 +1,19 @@
+# Line Ending Check Plugin
+
+This CiBuildPlugin scans all the files in a package to verify that the line endings are CRLF.
+
+## Configuration
+
+The plugin can be configured to ignore certain files.
+
+``` yaml
+"LineEndingCheck": {
+    "IgnoreFiles": []
+}
+```
+
+### IgnoreFiles
+
+An **optional** list of Unix style filename patterns used to exclude files from being checked.

```

Comment:
Is this still accurate?  Are we saying that .gitignore syntax is the same as unix style filename pattern?  Doesn't it also work with directories?  Should we mention, they must be relative from package root?

Reply:
Yes, the git ignore syntax is more robust. Along with the fnmatch link, I forgot to update this section. It's updated now.

https://github.com/microsoft/mu_basecore/pull/123/commits/7211bef603a41285dd60cb6de852f7cfb1b04e8c